### DISCUSSION TITLE
rog strix b550-a gaming compatibility issues?

### DISCUSSION THREAD

**Discussion by lynnthegrinch** [seerge/g-helper#1246] [Q&A]
hi, i've been trying to let ghelper run, and every time i have the same error.. "can't connect to ASUS ACPI. Application can't funcion without it. Try to install Asus System Control Interface.". I did reinstall the system control interface, both the v1 and v3, but it doesn't seem to make any difference. I even tried to run the scan of the pc to make sure the system files are not corrupted, and that does't seem to be the problem either.. does anyone have any idea what could it be? I've seen other people run ghelper with the b550-a already so i don't understand what to do. Thanks:)

**Comment by seerge** [comment]
@lynnthegrinch it's an app for laptops. Not desktop motherboards.

<img width="890" alt="Screenshot 2023-09-03 at 18 46 49" src="https://github.com/seerge/g-helper/assets/5920850/7887508e-dd94-4b2f-beaa-6790d77d902c">
